Services/TransactionService/Sources/TransactionService.swift

```diff
@@ -35,22 +35,37 @@ public struct TransactionService: Sendable {
 
     public func setup() {
         Task {
-            await runner.start(
-                every: .seconds(5),
-                runImmediately: true,
-                action: updatePendingTransactions
-            )
+            await poller.start(pollAction: pollPendingTransactions)
         }
     }
 
     public func addTransactions(walletId: String, transactions: [Transaction]) throws {
         try transactionStore.addTransactions(walletId: walletId, transactions: transactions)
+        setup()
```

Comment:
```suggestion
        setup()
```

why do we need to setup on adding new transaction?